### DISCUSSION TITLE
Landing page route?

### DISCUSSION THREAD

**Discussion by magickaito** [Budibase/budibase#6205] [Help] [answered]
How to control which route is the landing/home route?

Currently when my app loads, either when user logged in, or when you clicked on "view app" from builder, it always go to a detaile d view like: #/table/:id - which will then show "No rows found". 

I want to how the table listing as the default route, how can I do that?

**Comment by melohagan** [comment, marked as answer]
Hey @magickaito 

The default/home screen will always be the screen that has the path `/`

I've added a brief section to the docs. Thanks for highlighting! 
https://docs.budibase.com/docs/screens#home-screen

**Reply by magickaito** [reply]
cool thanks

**Reply by huining112** [reply]
@melohagan does this mean all users will have the same default landing screen? can admin/power users have a different role-based landing screen?

**Reply by melohagan** [reply]
Ah good question! Did a quick test there - the home screen is specific to the access level.
So you can have a homepage per access level.

E.g. if an admin user opens an app that has both a Basic and Admin level homepage, they will be shown the admin homepage, whereas a basic user would be shown the basic homepage.

**Reply by huining112** [reply]
@melohagan That would be very nice, maybe you could add a note in the documentation.

**Reply by melohagan** [reply]
@huining112 Already ahead of you ;) https://docs.budibase.com/docs/screens#home-screen
